content/_partners/faq._en.md

```diff
@@ -141,26 +141,26 @@ development_accordion:
         id: c-3
         title: How do we get our application to production?
         content: >-
-            <a href="https://developers.login.gov/production/" class="external-link">See our production deployment page here</a>. We deploy changes to our production configuration weekly on Thursdays between 11am – 2pm EST. 
+            <a href="https://developers.login.gov/production/" class="external-link">Check Production deployment for more details.</a> We deploy changes to our production configuration weekly on Thursdays between 11am – 2pm EST. 
     -
         id: c-4
         title: What do you support-SAML vs. OAuth vs. OpenID?
         content: >-
             SAML vs. OAuth vs. OpenID
 
-            - We do not support the OpenID Connect “implicit flow” with client_secret because it is <a href="https://oauth.net/2/grant-types/implicit/" class="external-link">not recommended by the OAuth group</a> for security reasons. We do support OpenID Connect private_key_jwt and PKCE
+            - We do not support the OpenID Connect “implicit flow” with <code class="highlighter-rouge">client_secret</code> because it is <a href="https://oauth.net/2/grant-types/implicit/" class="external-link">not recommended by the OAuth group</a> for security reasons. We do support OpenID Connect <code class="highlighter-rouge">private_key_jwt</code> and PKCE
```

Comment:
This gets markdown'ed so we could also potentitally use backticks here? I checked it out and it automatically adds`highligher-rouge` like this (and also `language-plaintext` but I don't think that matters much here)

We could also markdown the link just in case it reads more easily

```suggestion
            - We do not support the OpenID Connect “implicit flow” with `client_secret` because it is [not recommended by the OAuth group](https://oauth.net/2/grant-types/implicit/){:class="external-link"} for security reasons. We do support OpenID Connect `private_key_jwt` and PKCE
```